ui/decredmaterial/button.go

```diff
@@ -39,7 +45,19 @@ func Clickable(gtx layout.Context, button *widget.Clickable, w layout.Widget) la
 	return material.Clickable(gtx, button, w)
 }
 
-func (b Button) Layout(gtx layout.Context) layout.Dimensions {
+func (b *Button) SetIsDisabled(isDisabled bool) {
+	b.isDisabled = isDisabled
+}
+
+func (b *Button) IsDisabled() bool {
```

Comment:
similar to the Focused() method on the editor widget

```suggestion
func (b *Button) Enabled() bool {
```